doc/source/methods/Similarity.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>I do not remember exactly if they evaluated the <code>grad_asym_dot</code> . From what I remember they only used <code>grad-dot</code> and <code>grad-cos</code>. </p><p>Maybe is worth expanding a bit why <code>grad-dot</code> might perform a bit worst. Remember that the low performance of <code>grad-dot</code> is the sensitivity to outliers (i.e., instances which have large norms). <code>grad-cos</code> performs a normalisation which alleviates  this issue. You can find a nice description with an illustration  in the <a href="https://arxiv.org/pdf/2003.11630.pdf" target="_blank">Relative Influence Functions paper.</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I deleted the reference to grad_asym_dot. About the reasons, I thought about including it, but maybe it's too much for the docs. I linked the paper, so if someone if interested can go look it up. </p>